Upgrade to canteen-maven-plugin 1.0.1

joschi opened the pull request:
Refs salesforce/grpc-java-contrib#134
Fixes #223

rmichela commented:
First. Thanks so much for following up with this. I was hoping to fix https://github.com/salesforce/grpc-java-contrib/issues/141 before upgrading Canteen in reactive-grpc, so we don't have to do more than one release. 

How immediately do you need this released?

joschi commented:
> How immediately do you need this released?

No worries, you don't need to rush it. I have a workaround (adding the necessary environment variables to the `java` shim of asdf-vm) and currently don't rely on having it working out-of-the-box.

Thanks for checking this so quickly and keep up the good work!

joschi commented:
@rmichela Sorry for the ping. 😄 

Would it be possible to release this change in a patch release of `reactive-grpc`?

The changes since the last release look quite compact:
https://github.com/salesforce/reactive-grpc/compare/v1.0.1...master
